src/main/java/seedu/address/model/Model.java

```diff
@@ -76,6 +76,20 @@ public interface Model {
      */
     void setTask(Task target, Task editedTask);
 
+//    /**
+//     * Replaces the given task {@code target} with {@code markedTask}.
```

Comment:
Same with this

Reply:
Same reasoning as for ModelManager. Will remove in a future commit!